Tests/SwiftGenKitTests/ColorsTextFileTests.swift

```diff
@@ -6,47 +6,28 @@
 
 import XCTest
 import PathKit
-import SwiftGenKit
+@testable import SwiftGenKit
 
 class ColorsTextFileTests: XCTestCase {
-  func testEmpty() {
-    let parser = ColorsTextFileParser()
-
-    let result = parser.stencilContext()
-    XCTDiffContexts(result, expected: "empty.plist", sub: .colors)
-  }
-
-  func testListWithDefaults() throws {
-    let parser = ColorsTextFileParser()
-
-    try parser.addColor(named: "Text&Body Color", value: "0x999999")
-    try parser.addColor(named: "ArticleTitle", value: "#996600")
-    try parser.addColor(named: "ArticleBackground", value: "#ffcc0099")
-
-    let result = parser.stencilContext()
-    XCTDiffContexts(result, expected: "entries.plist", sub: .colors)
-  }
-
   func testFileWithDefaults() throws {
-    let parser = ColorsTextFileParser()
-    try parser.parseFile(at: Fixtures.path(for: "colors.txt", sub: .colors))
+    let parser = ColorsFileParser()
+    parser.colors = try ColorsTextFileParser().parseFile(at: Fixtures.path(for: "colors.txt", sub: .colors))
 
     let result = parser.stencilContext()
```

Comment:
Should we really test both the parsing AND the transformation into a StencilContext?

Shouldn't we instead just put an expectation directly on the dictionary returned by `parseFile` without transforming it into a stencilContext — and test that conversion of a dictionary into a stencilContext in a separate test suite?